src/manifest.jl

```diff
@@ -127,38 +127,45 @@ function validate_manifest(stage1::Dict{String,Vector{Stage1}})
                      "but entry with UUID `$uuid` has name `$(dep_entry.name)`.")
         end
     end
-    return manifest
+    return Manifest(; julia_version, manifest_format, deps)
 end
 
 function Manifest(raw::Dict)::Manifest
+    julia_version = isnothing(raw["julia_version"]) ? nothing : VersionNumber(raw["julia_version"])
+    manifest_format = VersionNumber(raw["manifest_format"])
+    if !in(manifest_format.major, 1:2)
+        @warn "Unknown Manifest.toml format version detected. Unexpected behavior may occur" manifest_format maxlog = 1
```

Comment:
Might want to give the path to the manifest as well.

Reply:
Fixed